python_modules/dagster/dagster_tests/core_tests/definitions_tests/test_composite_partitions.py

```diff
@@ -0,0 +1,39 @@
+from datetime import datetime
+
+from dagster import DailyPartitionsDefinition, StaticPartitionsDefinition
+from dagster._core.definitions.composite_partitions import CompositePartitionsDefinition
+
+DATE_FORMAT = "%Y-%m-%d"
+
+
+def test_composite_static_partitions():
```

Comment:
Seems like its possible in this implementation to create a composite partitions definition with other composite partitions definitions. Probably worth putting this under test